webgpu/lessons/webgpu-vertex-buffers.md

```diff
@@ -93,19 +93,21 @@ for `@location(0)`
 
 To the [`vertex`](GPUVertexState) entry of the [`pipeline` descriptor](GPURenderPipelineDescriptor) 
 we added a `buffers` array which is used to describe how to pull data out of 1 or more vertex buffers.
-For our first and only buffer, we set an `arrayStride` in number of bytes. a *stride* in this case is
+For our first and only buffer, we set an `arrayStride` in number of bytes. A *stride* in this case is
 how many bytes to get from the data for one vertex in the buffer, to the next vertex in the buffer.
 
 <div class="webgpu_center"><img src="resources/vertex-buffer-one.svg" style="width: 1024px;"></div>
 
 Since our data is `vec2f`, which is two float32 numbers, we set the
 `arrayStride` to 8.
 
-Next we define an array of attributes. We only have one. `shaderLocation: 0`
+Next we define an array of attributes. We only have one: `shaderLocation: 0`
 corresponds to `location(0)` in our `Vertex` struct. `offset: 0` says the data
 for this attribute starts at byte 0 in the vertex buffer. Finally `format:
 'float32x2'` says we want WebGPU to pull the data out of the buffer as two 32bit
-floating point numbers.
+floating point numbers. Note the `attributes` property is the same entity pictured in the 
+[simplified draw diagram](https://webgpufundamentals.org/webgpu/lessons/resources/webgpu-draw-diagram.svg) 
```

Comment:
I would prefer not to link to the svg file. Instead, in the target .md file, add an anchor with an id that starts with 'a-'. For example, in `webgpufundamentals.md`, 

```
<a id="a-webgpu-state-diagram"></a> Maybe a picture will help. ...
```

Then here, you can link to `[simplified draw diagram](webgpu-fundamentals.html#a-webgpu-state-diagram")`

This has a bunch of advantages:

1. Using an id that starts with `a-` means it's unlikely to conflict with auto-generated ids

2. It links to the explanatory text near to the diagram

3. It *should* work across language. Although in this particular case, you'd have to go add the same anchor in the existing fundamentals language .md files.

Reply:
used the existing anchor

[resolution commit](./154/commits/701f16582d2ae80c9efbf9927932e19531332e2f)